File: ordenacaoPython/vetores/vetores.py
import random

def list_create(tamanho):
    return [random.randint(1, 1000) for _ in range(tamanho)]

list_100 = list_create(100)
list_500 = list_create(500)
list_1k = list_create(1000)
list_5k = list_create(5000)
list_30k = list_create(30000)
list_80k = list_create(80000)
list_100k = list_create(100000)
list_150k = list_create(150000)
list_200k = list_create(200000)

File: algoritmos_ordenacao/vetores/Vetores.java
package algoritmos_ordenacao.vetores;

import java.util.Random;

public class Vetores {
    public static int[] criarVetorAleatorio(int tamanho) {
        int[] vetor = new int[tamanho];
        Random random = new Random();

        for (int i = 0; i < tamanho; i++) {
            vetor[i] = random.nextInt(1000);
        }

        return vetor;
    }

    public static int[] list_100() {
        return criarVetorAleatorio(100);
    }

    public static int[] list_500() {
        return criarVetorAleatorio(500);
    }

    public static int[] list_1k() {
        return criarVetorAleatorio(1000);
    }

    public static int[] list_5k() {
        return criarVetorAleatorio(5000);
    }

    public static int[] list_30k() {
        return criarVetorAleatorio(30000);
    }

    public static int[] list_80k() {
        return criarVetorAleatorio(80000);
    }

    public static int[] list_100k() {
        return criarVetorAleatorio(100000);
    }

    public static int[] list_150k() {
        return criarVetorAleatorio(150000);
    }

    public static int[] list_200k() {
        return criarVetorAleatorio(200000);
    }
}
